.github/workflows/publish.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Is it intentional that this only provides the latest version? Shouldn't this also create images for each tag so that you are not running on unstable versions?